packages/dialtone-vue2/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@dialpad/dialtone-emojis": "workspace:*",
     "@dialpad/dialtone-icons": "workspace:*",
+    "@dialpad/i18n-vue2": "1.0.2",
```

Comment:
Do we still need a token to access this or is it public now?

Reply:
That version is published to NPM, no token should be required, BUT I need to talk with @dlovero-dialpad to check if they're planning on keep publishing it to both GitHub and NPM so we can rely on that.

Also some configs like setting up the registry of `@dialpad` packages will make the token to be required to pull this package.